test/unit/sqs/index.test.ts

```diff
@@ -22,140 +23,296 @@ describe("SqsQueue tests", () => {
 		handle: jest.fn()
 	};
 
+	const mockErrorHandler = {
+		handle: jest.fn()
+	};
+
 	const generatePayload = (): TestMessage => ({ msg: uuidv4() });
 
-	const createSqsQueue = () => {
+	const createSqsQueue = (timeout: number) => {
 		return new SqsQueue({
 			queueName: TEST_QUEUE_NAME,
 			queueUrl: TEST_QUEUE_URL,
 			queueRegion: TEST_QUEUE_REGION,
-			longPollingIntervalSec: 0
+			longPollingIntervalSec: 0,
+			timeoutSec: timeout,
+			maxAttempts: 3
 		},
-		mockRequestHandler);
+		mockRequestHandler,
+		mockErrorHandler);
 	};
 
 	let queue: SqsQueue<TestMessage>;
 
-	beforeEach(() => {
-		queue = createSqsQueue();
-		queue.start();
-	});
+	describe("Normal execution tests", () => {
 
-	afterEach(async () => {
```

Comment:
Apologies for the refactoring here, but I had to group the existing tests under the `describe("Normal execution tests", ....`

Because timeouts test uses different before each/after each configuration